state/movie/operations/toggleToWatchlist.js

```diff
@@ -0,0 +1,33 @@
+import { createLogic } from 'redux-logic';
+
+import { getSessionId } from '../../session/selectors';
+import { TOGGLE_TO_WATCHLIST } from '../types';
+import { toggleToWatchlistSuccess } from '../../data/actions';
+
+const toggleToWatchlistLogic = createLogic({
+  type: TOGGLE_TO_WATCHLIST,
+
+  async process({ getState, httpClient, action }, dispatch, done) {
+    const { id, isMovieInWatchlist } = action;
+    const state = getState();
+    const sessionId = getSessionId(state);
+    const newStatus = !isMovieInWatchlist;
```

Comment:
It's better to send implicit action from component what logic should do, so change this value in the component, please

Reply:
Fixed